src/logger/logger.ts

```diff
@@ -1,8 +1,8 @@
 /*eslint-disable */
 import { MCLogger } from '@map-colonies/mc-logger';
-const service = require('../../package.json');
+const { name, version } = require('../../package.json');
 const LOGGER = (window as any)._env_.LOGGER;
 
-const logger : MCLogger = new MCLogger(LOGGER, service);
+const logger : MCLogger = new MCLogger(LOGGER, { name, version });
```

Comment:
where is the browser and it's version ? log time?

Reply:
Not here, line 28
Log time added as well
![image](https://user-images.githubusercontent.com/23166717/97168000-6ad4ea00-1790-11eb-993c-ad7d56cb3011.png)